CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ accidentally triggering the load of a previous DB version.**
 * #5547 Skip Ordered Append when only 1 child node is present
 * #5510 Propagate vacuum/analyze to compressed chunks
 * #5584 Reduce decompression during constraint checking
+* #5530 Optimize compressed chunk resorting
```

Comment:
This changelog entry was proposed here: https://github.com/timescale/timescaledb/pull/5530#issuecomment-1508443328 . The new entry would be too long for the commit / PR title. But I don't have a strong opinion on it. Maybe we can agree on a title in this thread.

Reply:
I think it can be different from commit title. Commit messages are for developers and contain technical details. And changelog messages are for users and describe the user-visible changes in behavior.

Reply:
I updated the changelog according to your suggestion. However, I have slightly modified your suggestion.